<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>E-commerce</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 246, 220)</string>
  </property>
  <widget class="QLabel" name="product_image">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>30</y>
     <width>241</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>6.jpg</pixmap>
   </property>
  </widget>
  <widget class="QTextEdit" name="product_desc">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>331</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(56, 53, 40);
font: 12pt &quot;Tw Cen MT&quot;;
color: rgb(43, 29, 4);
background-color: rgb(255, 253, 239);
border-width:2px;
border-radius:10px;
border-width:0.5px;
border-style:outset;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Description du produit</string>
   </property>
  </widget>
  <widget class="QLabel" name="product_name">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(43, 29, 4);
font: 12pt &quot;Tw Cen MT&quot;;</string>
   </property>
   <property name="text">
    <string>NAME</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="bidder_id">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(43, 29, 4);
font: 11pt &quot;Tw Cen MT&quot;;
border-color: rgb(43, 29, 4);
background-color: rgb(255, 253, 239);
border-width:2px;
border-radius:10px;
border-width:0.5px;
border-style:outset;</string>
   </property>
   <property name="placeholderText">
    <string>Entrez votre id</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="bidder_name">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(43, 29, 4);
font: 11pt &quot;Tw Cen MT&quot;;
border-color: rgb(43, 29, 4);
background-color: rgb(255, 253, 239);
border-width:2px;
border-radius:10px;
border-width:0.5px;
border-style:outset;</string>
   </property>
   <property name="placeholderText">
    <string>Entrez votre nom</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="bidder_price">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>310</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 253, 239);
font: 11pt &quot;Tw Cen MT&quot;;
border-color: rgb(43, 29, 4);
color: rgb(43, 29, 4);
border-width:2px;
border-radius:10px;
border-width:0.5px;
border-style:outset;</string>
   </property>
   <property name="suffix">
    <string> DA</string>
   </property>
  </widget>
  <widget class="QLabel" name="prix">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>290</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(43, 29, 4);
font: 11pt &quot;Tw Cen MT&quot;;</string>
   </property>
   <property name="text">
    <string>Prix  proposé</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bid_validation">
   <property name="geometry">
    <rect>
     <x>694</x>
     <y>310</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 219, 173);
border-color: rgb(43, 29, 4);
font: 13pt &quot;Tw Cen MT&quot;;
color: rgb(43, 29, 4);
border-width:2px;
border-radius:10px;
border-width:0.5px;
border-style:outset;</string>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QListView" name="product_list">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 253, 239);
font: 11pt &quot;Tw Cen MT&quot;;
border-color: rgb(43, 29, 4);
border-width:2px;
border-radius:10px;
border-width:0.5px;
border-style:outset;</string>
   </property>
  </widget>
  <widget class="QLabel" name="prix_estime">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>250</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(43, 29, 4);
font: 11pt &quot;Tw Cen MT&quot;;</string>
   </property>
   <property name="text">
    <string>Prix estimé : </string>
   </property>
  </widget>
  <widget class="QLabel" name="timer_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT</family>
     <pointsize>25</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);
font: 25pt &quot;Tw Cen MT&quot;;</string>
   </property>
   <property name="text">
    <string>30</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(43, 29, 4);
font: 14pt &quot;Tw Cen MT&quot;;</string>
   </property>
   <property name="text">
    <string>Timer</string>
   </property>
  </widget>
  <widget class="QListView" name="bidders_list">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>400</y>
     <width>561</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 253, 239);
font: 12pt &quot;Tw Cen MT&quot;;
color: rgb(43, 29, 4);
border-color: rgb(43, 29, 4);
border-width:2px;
border-radius:10px;
border-width:0.5px;
border-style:outset;</string>
   </property>
  </widget>
  <widget class="QLabel" name="sold">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>360</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(43, 29, 4);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
